test/e2e/mcd_test.go

```diff
@@ -1007,6 +1009,109 @@ func TestMCDRotatesCertsOnPausedPool(t *testing.T) {
 
 }
 
+func TestFirstBootHasSSHKeys(t *testing.T) {
+	cs := framework.NewClientSet("")
+	outNodeYoungest := bytes.NewBuffer([]byte{})
+	outErr := bytes.NewBuffer([]byte{})
+	// get nodes by newest
+	cmdCombined := "oc get nodes --sort-by .metadata.creationTimestamp | tail -n 1"
+	cmd := exec.Command("bash", "-c", cmdCombined)
+	cmd.Stdout = outNodeYoungest
+	cmd.Stderr = outErr
+	err := cmd.Run()
+	require.Nil(t, err, fmt.Sprintf("Got stdout: %s and stderr: %s", outNodeYoungest.String(), outErr))
+	// get top machineset
+	cmdCombined = "oc -n openshift-machine-api -o name get machinesets | head -n 1"
+	cmd = exec.Command("bash", "-c", cmdCombined)
+	outMSet := bytes.NewBuffer([]byte{})
+	outErr = bytes.NewBuffer([]byte{})
+	cmd.Stdout = outMSet
+	cmd.Stderr = outErr
+	err = cmd.Run()
+	require.Nil(t, err, fmt.Sprintf("Got stdout: %s and stderr: %s", outMSet.String(), outErr))
+	mset := strings.Trim(strings.Split(outMSet.String(), "/")[1], "\n")
+	// scale a 2nd machine
+	cmdCombined = "oc scale --replicas=2  machineset " + mset + " -n openshift-machine-api"
+	cmd = exec.Command("bash", "-c", cmdCombined)
+	outScale := bytes.NewBuffer([]byte{})
+	outErr = bytes.NewBuffer([]byte{})
+	cmd.Stdout = outMSet
+	cmd.Stderr = outErr
+	err = cmd.Run()
+	require.Nil(t, err, fmt.Sprintf("Got stdout: %s and stderr: %s", outScale.String(), outErr))
```

Comment:
question: Did you mean to wire up `outScale` to this error output instead of `outMSet`? I ask because you assigned `outMSet` to cmd.Stdout two lines just above here.

Reply:
yep, I'll fix this, thanks.